coalib/output/ConsoleInteraction.py

```diff
@@ -255,7 +255,8 @@ def print_lines(console_printer,
             console_printer.print('')
         else:
             console_printer.print(highlight_text(
-                no_color, line[printed_chars:], lexer), end='')
+                no_color, line[printed_chars:], lexer, BackgroundMessageStyle),
+                                  end='')
```

Comment:
Since logically we want to end the highlight at some point and not have it bled to the other lines.